ubuntu/python-conda/README.md

```diff
@@ -0,0 +1,8 @@
+# Conda Python Container
```

Comment:
Please include this important warning about Anaconda service agreement and get the links from here: https://docs.databricks.com/libraries/notebooks-python-libraries.html#manage-libraries-with-conda-commands


```
Important

Anaconda Inc. updated their terms of service for anaconda.org channels in September 2020. Based on the new terms of service you may require a commercial license if you rely on Anaconda’s packaging and distribution. See Anaconda Commercial Edition FAQ for more information. Your use of any Anaconda channels is governed by their terms of service.
```